### PR TITLE
Add some fields into the Assets entity

### LINKED ISSUE #277: Add some fields into the Assets entity
Dans le formulaire:
- ajout de l’immatriculation
- Ajout d'un champ textarea "Commentaires"

Dans le planning:
- Affichage de certaines informations au clic sur « plus d’informations »

### PR DESCRIPTION
**Pull request by nikophil**
fixes #277

### CODE REVIEW

File: src/Migrations/Version20200414132415.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+final class Version20200414132415 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Add "license_plate" and "comments" fields to "commissionable_asset"';
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
```
Comment (mRoca) [inline src/Migrations/Version20200414132415.php:22]: you can remove all the comments of this file

File: templates/organization/planning/_availabilities_assets.html.twig [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+{% block type %}{{ type }}{% endblock type %}
+
+{# columns number of blocks itemDataHeader and itemDataDetails should be the same than in _availabilities_users.html.twig #}
+{% block itemDataHeader %}
+    <th class="item-data">Radio mobile?</th>
+    <th class="item-data" colspan="2">Kit de premiers soins?</th>
+    <th class="item-data">Contact</th>
+    <th class="item-data" colspan="2">Lieu de stationnement</th>
+    <th class="item-data">Places</th>
+    <th class="item-data">Immatriculation</th>
```
Comment (fmeynard) [inline templates/organization/planning/_availabilities_assets.html.twig:12]: We should use translations here

File: templates/organization/planning/_availabilities_base.html.twig [hunk 3]
```diff
@@ -0,0 +1,51 @@
+<thead>
+<tr class="days">
+    <th rowspan="2"></th>
+    <th colspan="{{ itemDataRow }}" class="item-data">Informations</th>
```
Comment (fmeynard) [inline templates/organization/planning/_availabilities_base.html.twig:4]: translations

File: templates/organization/planning/_availabilities_assets.html.twig (hunk 2 above)
Comment (nikophil) [reply templates/organization/planning/_availabilities_assets.html.twig:12]: i'm waiting #289 to be merged

Comment (fmeynard) [reply templates/organization/planning/_availabilities_assets.html.twig:12]: Merge done, let me know if you want to address that in a separated PR

**Comment by nikophil** [comment]
@fmeynard translations OK, if you want to have a look, i've also added translations for assets' form

**Pull request merged**